test/verify/check-superuser

```diff
@@ -77,16 +80,16 @@ class TestSuperuser(MachineCase):
         b.leave_page()
         b.check_superuser_indicator("Administrative access")
 
+    @nondestructive
```

Comment:
Is there some particular reason why you left testTwoPasswds as destructive? This doesn't seem difficult either, and you could make the whole class nd. But, totally fine for a follow-up of course.